dev/workflow/applications/gefs.py

```diff
@@ -166,9 +170,6 @@ def get_task_names(self):
         if options['do_gempak']:
             tasks += ['gempak']
 
-        if options['nens'] > 0:
-            tasks += ['atmos_ensstat']
-
```

Comment:
I think it makes more sense to keep the `atmos_ensstat` job here (after products) since it is dependent on products.  The `awips` jobs should then be added in after `atmos_ensstat` at this point.

```suggestion
        if options['nens'] > 0:
            tasks += ['atmos_ensstat']
            if options['do_awips']:
                tasks += ['awips']
```